libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -251,16 +301,44 @@ MAV_RESULT AP_Camera::handle_command_long(const mavlink_command_long_t &packet)
         }
         return MAV_RESULT_DENIED;
     case MAV_CMD_IMAGE_START_CAPTURE:
-        if (!is_zero(packet.param2) || !is_equal(packet.param3, 1.0f) || !is_zero(packet.param4)) {
+        // currently we don't handle the parameter 4 i.e. sequence number
+        if (!is_zero(packet.param2) & !is_equal(packet.param3, 1.0f)) {
```

Comment:
I actually don't think we should bother checking the interval.  It's a multiple picture request if the total number of images is <0 or >1.  By the way we should use "&&".  "&" is a binary and.